codegen/tests/simple/skeletonhandlerimpl.go

```diff
@@ -0,0 +1,58 @@
+// Code generated by sysl DO NOT EDIT.
+package simple
```

Comment:
What's the benefit of generating this file? Custom code doesn't need to provide a handler for every endpoint and every handler they do provide will be custom.

Reply:
will be removed, its not needed anymore.